gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -467,14 +496,15 @@ gst_tensor_query_client_sink_event (GstPad * pad,
     case GST_EVENT_CAPS:
     {
       GstCaps *caps;
-      gchar *_topic = NULL, *protocol_str = NULL;
+      gchar *_topic = NULL;
```

Comment:
IMO we dont need topic for tcp connection.

Reply:
Agree! Updated.